schemas/manifest.schema.json

```diff
@@ -0,0 +1,473 @@
+                    "additionalProperties": false,
+                    "patternProperties": {
+                        "^[a-z][a-z0-9_]*$": {
+                            "type": "object",
+                            "required": [],
+                            "additionalProperties": false,
+                            "properties": {
+                                "url": {
+                                    "type": "string",
+                                    "format": "uri"
+                                },
+                                "sha256": {
+                                    "type": "string",
+                                    "pattern": "[0-9a-z]{64}"
+                                },
+                                "in_subdir": {
+                                    "type": "boolean"
+                                },
+                                "prefetch": {
+                                    "type": "boolean"
+                                },
+                                "format": {
+                                    "type": "string",
+                                    "enum": [
+                                        "tar.gz",
+                                        "tar.xz",
+                                        "tar.bz2",
+                                        "zip",
+                                        "docker",
+                                        "whatever"
+                                    ]
+                                },
+                                "extract": {
+                                    "type": "boolean"
+                                },
+                                "rename": {
+                                    "type": "string"
+                                },
+                                "platform": {
+                                    "type": "string"
+                                },
+                                "autoupdate": {
+                                    "type": "object",
+                                    "required": [],
+                                    "additionalProperties": false,
+                                    "properties": {
+                                        "strategy": {
+                                            "type": "string",
+                                            "enum": [
+                                                "latest_github_tag",
+                                                "latest_github_release",
+                                                "latest_github_commit"
+                                            ]
+                                        },
+                                        "upstream": {
+                                            "type": "string"
+                                        },
+                                        "asset": {
+                                            "oneOf": [
```

Comment:
In addition to `strategy`, `upstream` and `asset`, we can have a `force_version` arg of type string, cf for example https://github.com/YunoHost-Apps/indexhibit_ynh/blob/master/manifest.toml#L45

Reply:
fixed